code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -26,6 +27,13 @@
 /obj/item/weapon/reagent_containers/borghypo/lost
 	reagent_ids = list("tricordrazine", "bicaridine", "dexalin", "anti_toxin", "tramadol", "spaceacillin")
 
+/obj/item/weapon/reagent_containers/borghypo/merc
+	name = "advanced cyborg hypospray"
+	desc = "An advanced nanite and chemical synthesizer and injection system, designed for heavy-duty medical equipment.  This type is capable of safely bypassing \
+	thick materials that other hyposprays would struggle with."
+	bypass_protection = TRUE // Because mercs tend to be in spacesuits.
+	reagent_ids = list("healing_nanites", "iron", "hyperzine", "tramadol", "oxycodone", "spaceacillin", "peridaxon", "osteodaxon", "myelamine")

```

Comment:
Does injecting Iron work? I thought it needed to be ingested.

Reply:
I actually don't know.  Blood levels appeared to rise when injecting but it might've been the base regen rate.

Reply:
Apparently it's ingest only.  Guess I'll remove it.

Reply:
I'll look into making a replacement Chem this weekend.